ecommerce_integrations/shopify/invoice.py

```diff
@@ -63,13 +63,23 @@ def set_cost_center(items, cost_center):
 		item.cost_center = cost_center
 
 
-def make_payament_entry_against_sales_invoice(doc, setting, posting_date=None):
+def make_payament_entry_against_sales_invoice(doc, setting, shopify_order, posting_date=None):
 	from erpnext.accounts.doctype.payment_entry.payment_entry import get_payment_entry
+	_bank_account = setting.get('cash_bank_account')
 
-	payment_entry = get_payment_entry(doc.doctype, doc.name, bank_account=setting.cash_bank_account)
+	if len(shopify_order.get('payment_gateway_names')):
```

Comment:
```suggestion
	if len(shopify_order and shopify_order.get('payment_gateway_names') or []):
```

Handle case when it's not sent.